macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -434,36 +471,71 @@ fileprivate extension Unicode.Scalar {
 	static let zeroWidthSpace = Unicode.Scalar(0x200B)!
 }
 
-fileprivate extension NSColor {
+extension NSColor {
```

Comment:
I think this shouldn't be an extension of NSColor anymore.
Semantically it belongs to the AvatarView class.